internals/daemon/daemon_test.go

```diff
@@ -1176,3 +1177,136 @@ services:
 	c.Assert(tasks, HasLen, 1)
 	c.Check(tasks[0].Kind(), Equals, "stop")
 }
+
+type rebootSuite struct{}
+
+var _ = Suite(&rebootSuite{})
+
+func mockSyncSyscall(f func()) (restore func()) {
```

Comment:
mock -> fake

Reply:
Yeah, and should be made "public" in the export_test.go package (FakeSyncSyscall...)

Reply:
Fixed.